### DISCUSSION TITLE
What about incoming connections

### DISCUSSION THREAD

**Discussion by stepanov1975** [haugene/docker-transmission-openvpn#2150] [Q&A]
First of all thanks for the very nice and useful container with good explanations on how to use it.
Now, what about incoming connections? As I understand OpenVPN is used for downloads. But how can I allow uploads? Do i need to create port forwarding from the router? Or is it impossible when using VPN?

**Comment by pkishino** [comment]
? Why would it only be for downloads? Are you talking about seeding here? It should work right out of the box. Depending on VPN  provider you might need to set the peer port to a specific port and/or do something in their webui etc but otherwise nothing more

**Reply by stepanov1975** [reply]
@pkishino Thank you for the replay. Yes, I am talking about seeding. So you mean VPN provider should support port forwarding? As I understand NordVPN doesn't support port forwarding (and that is what I am using now). I think documentation need to have section about seeding
